plugins/source/github/client/client.go

```diff
@@ -233,7 +233,7 @@ func githubClientForHTTPClient(httpClient *http.Client, logger zerolog.Logger, e
 	}
 
 	if enterpriseSettings != nil {
-		return github.NewEnterpriseClient(enterpriseSettings.BaseURL, enterpriseSettings.UploadURL, rateLimiter)
```

Comment:
`NewEnterpriseClient` has been deprecated and throws errors when running `make lint`, the replacement is to use `WithEnterpriseURLs`